src/grafana_dashboards/postgresql-metrics.json

```diff
@@ -2092,7 +2103,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "pg_stat_database_tup_inserted{datname=~\"$datname\", instance=~\"$instance\"} != 0",
+          "expr": "irate(pg_stat_database_tup_returned{datname=~\"$datname\", instance=~\"$instance\"}[5m])",
```

Comment:
to represent the number/rate of `SELECT`-type operations, I opted to use `tup_returned` instead of `tup_fetched` (which is still being shown on the graph, just under a different panel named ` read count/rate (fetched)`. They are slightly different metrics, theres a good explanation here: https://www.datadoghq.com/blog/postgresql-monitoring/#read-query-throughput-and-performance